### ISSUE TITLE
[V2] Is there a way to change user lang when offline 

### ISSUE THREAD

**Issue by almogdesign** [gadicc/meteor-messageformat#126]
I wondering if its possible to change the local setting in offline and will it still update the user lang setting? (which is pure server side code) Is there a possible workaround?

**Comment by gadicc** [comment]
This isn't currently supported.  It might just work if you use GroundedDB on Meteor.users collection.  Otherwise I think we could probably get this working, and this would be a useful feature.

**Comment by almogdesign** [comment]
Yes working in staging and production not sure why it didn't work on my local host m
